src/components/common/FiatValue/index.tsx

```diff
@@ -18,10 +26,27 @@ const FiatValue = ({ value, maxLength }: { value: string | number; maxLength?: n
     return formatCurrencyPrecise(value, currency)
   }, [value, currency])
 
+  const [whole, decimals, endCurrency] = useMemo(() => {
+    const match = preciseFiat.match(/(.+)(\D\d+)(\D+)?$/)
+    return match ? match.slice(1) : ['', preciseFiat, '', '']
+  }, [preciseFiat])
+
   return (
-    <Tooltip title={preciseFiat}>
+    <Tooltip title={precise ? undefined : preciseFiat}>
       <span suppressHydrationWarning style={style}>
-        {fiat}
+        {precise ? (
+          <>
+            {whole}
+            {decimals && (
+              <Typography component="span" color="text.secondary" fontSize="inherit">
+                {decimals}
+              </Typography>
```

Comment:
```suggestion
              <Typography component="span" color="text.secondary" fontSize="inherit" fontWeight="bold">
                {decimals}
              </Typography>
```

According to the Figma design it should be bold: https://www.figma.com/design/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=10805-28960&t=fxUQpHt0EkGqo8i4-0